code/datums/components/lockon_aiming.dm

```diff
@@ -178,7 +178,7 @@
 
 /datum/component/lockon_aiming/proc/can_target(atom/A)
 	var/mob/M = A
-	return is_type_in_typecache(A, target_typecache) && !(ismob(A) && mob_stat_check && M.stat != CONSCIOUS) && !immune_weakrefs[WEAKREF(A)]
+	return is_type_in_typecache(A, target_typecache) && !(ismob(A) && mob_stat_check && M.stat != (CONSCIOUS || SOFT_CRIT)) && !immune_weakrefs[WEAKREF(A)]
```

Comment:
need to use 
```dm
stat != CONSCIOUS && stat != SOFT_CRIT
```
this won't work while the target is actually conscious because `(CONSCIOUS || SOFT_CRIT)` will always be truthy.

needs replacing everywhere this check is made

Reply:
addressed above